_workinggroups/softwaredevelopertools.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Software/Developer Tools 
+layout: plain
+redirect_from:
+  - /activities/softwaredevelopertools.html
+---
+
+The Software/Developer Tools Working Group aims to help HEP community in developing the latest and modern feature-rich projects
+with the help of software tools such as code editors, static/dynamic code analyzers, compilers, debuggers, performance analyzers and so on.
+The group plans to accomplish this by mainly serving as a forum where developers can share their experiences, discuss new/existing tools they utilize,
+and learn from each other. To that point, contributions from the entire community are strongly encouraged.
+
+The group is currently coordinated by Giulio Eulisse (CERN, EP-AIP-SDS), Alaettin Serhan Mete (UC Irvine, ATLAS), and Martin Ritter (LMU, Belle II).
+All coordinators can be reached at <hsf-tech-forum@googlegroups.com>.  
```

Comment:
That's a bad way to reach the coordinators! About 50 people are on that list.

I would suggest you either give your individual email addresses, or setup a coordinators email list to reach only you three.

Reply:
Done.